src/layerelement.cpp

```diff
@@ -1642,6 +1642,18 @@ std::pair<int, bool> LayerElement::CalcElementHorizontalOverlap(Doc *doc,
                     - HorizontalRightOverlap(otherElements.at(i), doc, horizontalMargin - shift, verticalMargin);
             }
         }
+        else if (Note *currentNote = vrv_cast<Note *>(this);
+                 Is(NOTE) && (currentNote->GetDrawingDur() == DUR_1) && otherElements.at(i)->Is(STEM)) {
```

Comment:
It think it would be clearer to do
```cpp
else if (this->Is(NOTE)){
    Note *currentNote = vrv_cast<Note *>(this);
    assert(note);
    if ((currentNote->GetDrawingDur() == DUR_1) && otherElements.at(i)->Is(STEM)) {
    //...
```